config/controller.yaml

```diff
@@ -65,12 +65,25 @@ spec:
         app: kpack-controller
         version: #@ data.values.version
     spec:
+      securityContext:
```

Comment:
Do we expect to remove the overlays in the corresponding package for kpack? https://github.com/vmware-tanzu/package-for-kpack/blob/main/config/overlay.yaml#L8